static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -118,6 +118,7 @@ define([
             if (guardian.isEnhanced) {
                 ab.segmentUser();
                 ab.run();
+                ab.trackEvent();
```

Comment:
Guess so, I wasn't sure if run meant “run the tests” or “initialise this
module”.

On Fri, 1 Apr 2016 at 16:45 Sam Desborough <notifications@github.com> wrote:

> In static/src/javascripts/bootstraps/standard/main.js
> <https://github.com/guardian/frontend/pull/12393#discussion_r58224743>:
>
> > @@ -118,6 +118,7 @@ define([
> >              if (guardian.isEnhanced) {
> >                  ab.segmentUser();
> >                  ab.run();
> > +                ab.trackEvent();
>
> I don't think so - segmenting users, running tests, and firing the
> tracking event are distinct operations.
> Would there be any benefit in conflating it with run?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/12393/files/1775a7dbaf7c49a18f4caece9adf8e9861501a23#r58224743>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396